test/fixtures.py

```diff
@@ -39,6 +39,10 @@
 from typeagent.storage.memory import MemoryStorageProvider
 from typeagent.storage import SqliteStorageProvider
 
+from openai.types.create_embedding_response import CreateEmbeddingResponse, Usage
+from openai.types.embedding import Embedding
+import tiktoken
+
```

Comment:
Please move this block before the first typeagent import (line 12).

I like the following order for imports:
1. stdlib
2. 3rd party deps
3. deps from this project

And alphabetized by fully qualified module name within each section.